packs/feats/charming-liar.json

```diff
@@ -28,7 +28,20 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
+                "outcome": [
+                    "criticalSuccess"
+                ],
+                "predicate": [
+                    "action:lie"
+                ],
+                "selector": "skill-check",
```

Comment:
Should the selector be `deception`?

Reply:
Strangely, Charming Liar doesn't mention the Lie actions it benefits are solely using Deception. If a PC had a way to replace Deception with another skill for the Lie Action, Charming Liar would still apply.